centreon/packages/ui/src/Graph/Gauge/useResizeObserver.ts

```diff
@@ -0,0 +1,68 @@
+import {
```

Comment:
there is already a resizeObserver in centreon/ui
do we use this or the old one, ore the one here only useful for Gauge
![image](https://github.com/user-attachments/assets/597b7829-17e4-46dc-9fe1-01f2e5da2a98)

Reply:
This one will be replaced because it's uselessly complicated